app/controllers/jobseekers/confirmations_controller.rb

```diff
@@ -1,15 +1,15 @@
 class Jobseekers::ConfirmationsController < Devise::ConfirmationsController
   def show
-    super if request.method == "POST"
+    super
   end
 
   protected
 
-  def after_confirmation_path_for(resource_name, resource)
+  def after_confirmation_path_for(_resource_name, resource)
```

Comment:
We now want to redirect to the confirmation_jobseekers_account page every time that a user confirms their account, I double checked this behaviour with Alex to make sure!